### PR TITLE
Update link to the bundle download

### PR DESCRIPTION
**Pull request by ItielBeeri**
Probably you would like to use a forward link (which I don't know how to achieve/generate); anyway, the link should be updated.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  please review. We probably should use a fwlink.

File: aspnetcore/publishing/iis.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Proceed through the **Confirmation** step to install the web server role and ser
...
+1. Install the [.NET Core Windows Server Hosting](https://download.microsoft.com/download/C/3/2/C32D45DC-6057-4E09-8FE2-25416934BDBB/DotNetCore.1.0.5_1.1.2-WindowsHosting.exe) bundle on the hosting system. The bundle will install the .NET Core Runtime, .NET Core Library, and the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module). The module creates the reverse-proxy between IIS and the Kestrel server. Note: If the system doesn't have an Internet connection, obtain and install the *[Microsoft Visual C++ 2015 Redistributable](https://www.microsoft.com/download/details.aspx?id=53840)* before installing the .NET Core Windows Server Hosting bundle.
```
Comment (tdykstra) [inline aspnetcore/publishing/iis.md]: We should either change the fwlink target (if the intent for this fwlink is to represent the current download) or make a new fwlink and use that here (if we want an fwlink for each version).  The former option makes more sense to me.

**Comment by GuardRex** [comment]
@tdykstra A customer asked about the 1.0.5/1.1.2 version of the bundle link on Twitter. @leecow Is there a new fwlink for it? ...or is the plan to target the current one (844461) to the latest? ... it's still pulling down the 1.0.4/1.1.1 bundle.

**Comment by leecow** [comment]
@GuardRex - https://go.microsoft.com/fwlink/?linkid=848766 points to DotNetCore.1.0.5_1.1.2-WindowsHosting.exe.

**Comment by GuardRex** [comment]
@leecow Thanks.

@tdykstra You want me to drop that in? @Rick-Anderson was going to do it, but I have time now to shoot it in there if you want.

**Comment by tdykstra** [comment]
Feel free to take care of it!
________________________________

**Comment by GuardRex** [comment]
I read your mind. 🔮 :grinning:.

**Comment by ItielBeeri** [comment]
@GuardRex thanks for the efficient handling. I close this.

**Pull request closed without merging**